src/test/java/seedu/sellsavvy/storage/JsonAdaptedOrderTest.java

```diff
@@ -11,15 +11,18 @@
 import seedu.sellsavvy.model.order.Count;
 import seedu.sellsavvy.model.order.Date;
 import seedu.sellsavvy.model.order.Item;
+import seedu.sellsavvy.model.order.Status;
 
 public class JsonAdaptedOrderTest {
     private static final String INVALID_ITEM = " ";
     private static final String INVALID_COUNT = "0";
     private static final String INVALID_DATE = "2-2-2";
+    private static final String INVALID_STATUS = "not pending";
 
     private static final String VALID_ITEM = BLOCKS.getItem().toString();
     private static final String VALID_COUNT = BLOCKS.getCount().toString();
     private static final String VALID_DATE = BLOCKS.getDate().toString();
+    private static final String VALID_STATUS = BLOCKS.getStatus().toString();
```

Comment:
Is there really no success test cases or you forgot to edit?

Reply:
![image](https://github.com/user-attachments/assets/0f1bf699-5f59-47e3-9efb-8d47ab40242c)
this is the success test case which has the original valid values of `BLOCKS`